storage/store.go

```diff
@@ -682,6 +715,9 @@ func (s *Store) Gossip() *gossip.Gossip { return s.gossip }
 // SplitQueue accessor.
 func (s *Store) SplitQueue() *splitQueue { return s.splitQueue }
 
+// ReplicateQueue accessor.
```

Comment:
We won't need this accessor if we move the loop to up-replicate into the replicate queue.